docs/Catalog/Preview.md

```diff
@@ -29,14 +29,28 @@ The following file formats are supported:
 * .ent
 * .pdb
 
+## Image previews
+
+The Quilt Catalog uses a [Lambda
+function](https://github.com/quiltdata/quilt/tree/master/lambdas/thumbnail)
+to automatically generate thumbnail previews of common image formats
+and select microscopy image formats such as .gif, .jpg, .png, .tif, .tiff
+(including `OME-TIFF`), and .czi.
```

Comment:
I don't know if it is worth listing all extensions, but FYI we also support .jpeg, .webp, .bmp